client/src/api/api-client.hook.ts

```diff
@@ -67,9 +68,11 @@ export const useApiClient = () => {
         // return Promise.reject({ message: 'Please re-authenticate' });
       }
 
-      if (!response.ok) {
-        return Promise.reject({ message: 'Please re-authenticate' });
-      }
+      // 'response.ok', response.ok);
+      // if (!response.ok) {
+      //   console.log('response in apiClient', response);
+      //   return Promise.reject({ message: 'Please re-authenticate' });
+      // }
```

Comment:
Why have you commented this out? I'm not saying your wrong, just trying to understand why.